beginner_source/blitz/autograd_tutorial.py

```diff
@@ -2,6 +2,12 @@
 """
 A Gentle Introduction to ``torch.autograd``
 ---------------------------------
+**Video by**: `Elliot Waite <https://www.youtube.com/@elliotwaite>`_
```

Comment:
Can we please not embed a video as we generally only embed videos from the official PyTorch youtube channel. Please add a link under **Further readings** on line 319 at the bottom of this tutorial. Thank you for understanding!

Reply:
Certainly, I've updated the PR. Let me know if I should make any changes to the link text.